charts/psmdb-db/production-values.yaml

```diff
@@ -18,6 +21,9 @@ fullnameOverride: ""
 pause: false
 unmanaged: false
 allowUnsafeConfigurations: false
+multiCluster:
+  enabled: false
+  # DNSSuffix: svc.clusterset.local
```

Comment:
if `enabled: false` we can uncomment `DNSSuffix: svc.clusterset.local` I suppose.

Reply:
My understanding is that these values are mostly for some sane minimum defaults and some commented examples, so I try to set as less defaults as possible.